docs/generated/changelog.html

```diff
@@ -22,6 +22,7 @@ <h2>Version 0.10.5</h2>
           of logging the user out and prompting re-authentication. See the @dfinity/auth-client
           Readme for more details
         </li>
+        <li>Fix for versioning tool</li>
```

Comment:
Is this an accurate description of what was fixed?
```suggestion
        <li>Versioning tool now sets patch version to 0 for minor version updates, or patch and minor versions to 0 for major version updates</li>
```